README.md

```diff
@@ -1,318 +1,403 @@
-# Slack Send GitHub Action
+# Slack Send
+
+> the GitHub Action for sending data to Slack
 
 [![codecov](https://codecov.io/gh/slackapi/slack-github-action/graph/badge.svg?token=OZNX7FHN78)](https://codecov.io/gh/slackapi/slack-github-action)
 
-Send data into Slack using this GitHub Action!
+Send data into Slack with a Slack [API method](#technique-2-slack-api-method)
```

Comment:
I studied the current `README` more and think it pairs well with your suggestion! A bulleted list is much more helpful for a quick introduction or skim 🙏

This first section was updated to link to details of the "Sending techniques" section and other configurations instead of attempting some quick summarization 😭